snuba/datasets/configuration/transactions/dataset.yaml

```diff
@@ -0,0 +1,8 @@
+version: v1
+kind: dataset
+name: transactions
+
+is_experimental: false
+
+entities:
+  all: [transactions]
```

Comment:
Could you elaborate what the `all` parameter does here and why it is needed?

Reply:
From my understanding, there is no functional reason why this is here other than to map closer to the Python definition of Dataset? I think we can remove this.

Reply:
I think you can list the entities themselves, without the `all` keyword:

```
entities:
- transactions
- ...
```